### ISSUE TITLE
Trying.. and not working in Magento 1.9 :(

### ISSUE THREAD

**Issue by Crustymd** [Vinai/VinaiKopp_StoreUrlRewrites#5]

**Comment by Vinai** [comment]
If you expect assistance please provide some meaningfull explanatiion of what you have tried and what is not working.

**Comment by hiLaura** [comment]
I tried on magento 1.9 too. I just copied Vinai files to my webserver as told and then I went to create a new category with different store views. I took out the 'ckeck' in 'Create Permanent Redirect for old URL' and saved it. I refreshed, reindexed and flushed everything, but anyway - when I switch language, the url is the same old and I get 404. :/
I hope the modman goes in the same folder as app folder..
